src/integrationTest/resources/TestProject/build.gradle

```diff
@@ -3,16 +3,11 @@ buildscript {
     maven {
       url uri('libs')
     }
-    //flatDir dirs: "C:/Users/i_am_/workspace/umple.gradle/build/libs"
-    maven {
-      url "https://plugins.gradle.org/m2/"
-    }
   }
   dependencies {
-    // Use the umple jar from the main umple.gradle project
     classpath files('libs/umple-latest.jar')
-    classpath "gradle.plugin.cruise.umple:UmpleGradlePlugin:0.1.2"
-    //classpath "umple.gradle.plugin:UmpleGradlePlugin-0.1.2"
+    // Use the locally built version of the plugin for the tests
+    classpath files('libs/UmpleGradlePlugin-0.1.3.jar')
```

Comment:
I'm still not sure about this. I can add `TestProject` as a child project of the root, but I can't set an `ext` field or anything from inside `settings.gradle`. In fact, the kinds of project settings I change change are quite limited: https://docs.gradle.org/current/javadoc/org/gradle/api/initialization/ProjectDescriptor.html

Pretty much just the name of the build file, the project name, etc. I don't see how I can add a version number. I also don't like the idea of making `TestProject` a sub project of the gradle plugin project. I feel that a sub project should be directly involved in the build somehow. `TestProject` is only used in integration testing.